code/modules/vore/fluffstuff/custom_items_vr.dm

```diff
@@ -892,6 +892,14 @@ obj/item/weapon/material/hatchet/tacknife/combatknife/fluff/katarina/handle_shie
 		accessset = 1
 	..()
 
+//verkister: Cameron Eggbert - Science goggles that ACTUALLY do nothing.
+/obj/item/clothing/glasses/science_proper
+	name = "Aesthetic Science Goggles"
+	desc = "The goggles really do nothing this time!"
```

Comment:
I assume this is to avoid the purple overlay?

Regardless, the forum thread you linked, http://forum.vore-station.net/viewtopic.php?f=26&t=957&sid=e4bfb9a1ce015f3e3ab282d998263a68 is not related to this item. Wrong post?